DorisProject/src/app/pages/stock/stock.component.ts

```diff
@@ -12,8 +14,10 @@ export class StockComponent implements OnInit {
 
   products: any[] = [];
   productsMaster: any[] = [];
+  responseModal: any;
 
-  constructor(private productService: ProductService, private  router: Router) {
+  constructor(private productService: ProductService, private  router: Router,
```

Comment:
Codacy has a fix for the issue: [Replace `private·productService:·ProductService,·private·` with `⏎····private·productService:·ProductService,⏎····private`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9258681)
```suggestion
  constructor(
```